lib/transform.js

```diff
@@ -72,14 +72,24 @@ fs.readdirSync(exampleFolder)
       const json = csdl.xml2json(xml, { strict: true });
       fs.writeFileSync(
         exampleFolder + example + ".json",
-        JSON.stringify(json, null, 4)
+        JSON.stringify(json, null, 4),
       );
+      if (!example.endsWith("-sample")) {
```

Comment:
Why not use a suffix such as `-vocabsample` to trigger vocab-style markdown conversion? The vocab-style markdown conversion is the new exception, so make it an exception and not the new default.